articles/support/pre-deployment/how-to-run-test.md

```diff
@@ -0,0 +1,56 @@
+---
+description: How to run the Auth0 Pre-Deployment Tests to ensure that your Clients are production-ready
+---
+
+# How to Run the Pre-Deployment Test
+
+The Pre-Deployment Tests are available via the [Auth0 Support Center](https://support.auth0.com/).
```

Comment:
I'm also thinking maybe we should add a note at the beginning of each doc saying that the feature is available only to paid customers (plus those in trial I think)? You can link here for details https://auth0.com/docs/support